src/main/java/co/omise/models/Refund.java

```diff
@@ -147,8 +147,7 @@ protected HttpUrl path() {
             if (options == null) {
                 options = new ScopedList.Options();
             }
-
-            return buildUrl(Endpoint.API, "charges", chargeId, "refunds");
+            return new HttpUrlBuilder(Endpoint.API, "charges", serializer()).segments(chargeId, "refunds").params(options).build();
         }
```

Comment:
Can you please put these on different lines?

Reply:
Ok, I'll change this.